docs/source/kubernetes/gettingstarted.rst

```diff
@@ -0,0 +1,112 @@
+.. _kubernetes_gettingstarted:
+
+Getting Started (Alpha)
+=======================
+
+The Kubernetes Provider (Alpha) is an optional package which an be installed to enable writing
+policies which interact with Kubernetes related resources.
+
+
+.. kubernetes_install-cc:
+
+Install Kubernetes Plugin
+-------------------------
+
+First, ensure you have :ref:`installed the base Cloud Custodian application
+<install-cc>`. Cloud Custodian is a Python application and must run on an
+`actively supported <https://devguide.python.org/#status-of-python-branches>`_
+version. 
+
+Once the base install is complete, you are now ready to install the Kubernetes provider package
+using one of the following options:
+
+Option 1: Install released packages to local Python Environment
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: bash
+
+    pip install c7n
+    pip install c7n_kube
+
+
+Option 2: Install latest from the repository
+"""""""""""""""""""""""""""""""""""""""""""""
+
+.. code-block:: bash
+
+    git clone https://github.com/cloud-custodian/cloud-custodian.git
+    pip install -e ./cloud-custodian
+    pip install -e ./cloud-custodian/tools/c7n_kube
+
+.. _kubernetes_authenticate:
+
+Connecting to your Cluster
+--------------------------
+
+The Custodian Kubernetes provider automatically uses your kubectl configuration or the config
+file set by the environment variable ``KUBECONFIG``. See the `Kubernetes Docs <https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/>`_
+for more information.
+
+.. _kube_write-policy:
+
+Write Your First Policy
+-----------------------
+A policy is the primary way that Custodian is configured to manage cloud resources.
+It is a YAML file that follows a predetermined schema to describe what you want
+Custodian to do.
+
+There are three main components to a policy:
+
+* Resource: the type of resource to run the policy against
+* Filters: criteria to produce a specific subset of resources
+* Actions: directives to take on the filtered set of resources
+
+In the example below, we will write a policy that filters for pods with a label "custodian"
+and deletes it:
+
+Filename: ``custodian.yml``
+
+.. code-block:: yaml
+
+    policies:
+      - name: my-first-policy
+        description: |
+          Deletes pods with label name:custodian
+        resource: k8s.pod
+        filters:
+          - type: value
+            key: metadata.labels.name
+            value: custodian
+        actions:
+          - type: delete
+
+.. _kube_run-policy:
+
+Run Your Policy
+---------------
+First, ensure you have :ref:`configured connectivity to your cluster <kubernetes_authenticate>`.
+
+Next, run the following command to execute the policy with Custodian:
+
+.. code-block:: bash
+
+   custodian run --output-dir=output custodian.yml --cache-period 0 -v
+
+If successful, you should see output similar to the following on the command line::
```

Comment:
These docs are a little weird because they are assuming you have a `pod` labeled with `custodian` but we don't walk them through creating anything.   Maybe we could add something like:

```
❯ kubectl run nginx --image=nginx --labels=name=custodian
❯ kubectl get pod -o wide --show-labels
NAME    READY   STATUS    RESTARTS   AGE   IP           NODE     NOMINATED NODE   READINESS GATES   LABELS
nginx   1/1     Running   0          24s   10.0.1.224   worker   <none>           <none>            name=custodian
```

Then show them running the policy and having it disappear?

Reply:
i moved the docs out to a separate branch for now